core/src/main/java/io/grpc/transport/ClientTransport.java

```diff
@@ -63,7 +63,7 @@ ClientStream newStream(MethodDescriptor<?, ?> method,
                          ClientStreamListener listener);
 
   /**
-   * Starts transport. Implementations must not call {@code listener} until after {@code start()}
+   * Starts transport. Implementations must not call {@code listener} until after {@link #start}
```

Comment:
Cross-references doesn't make sense here though, since it is referring to itself.

I do think it was a conscious decision to use code instead of link. The fact it is code also reduces the noise of the JavaDoc. But whatever.